src/Advection/centered_reconstruction.jl

```diff
@@ -61,7 +61,7 @@ Base.show(io::IO, a::Centered{N, FT, XT, YT, ZT}) where {N, FT, XT, YT, ZT} =
     print(io, summary(a), " \n",
               " Boundary scheme : ", "\n",
               "    └── ", summary(a.boundary_scheme), "\n",
-              " Directions:", "\n",
+              " Directions :", "\n",
```

Comment:
why the space?

Reply:
ah, just because also the other entries had it, we can remove it

Reply:
I guess my questions was for all other entries as well...